servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/InsufficientlySizedExecutorHttpTest.java

```diff
@@ -116,6 +116,7 @@ public void insufficientServerCapacityStreaming() throws Exception {
         insufficientServerCapacityStreaming0();
     }
 
+    // TODO Windows seeing CHANNEL_CLOSED_INBOUND. Is 1 thread enough with control events (e.g. close)?
```

Comment:
Consider creating an issue, otherwise it's hard to track.

Reply:
actually one already exists :) https://github.com/apple/servicetalk/issues/336

Reply:
I'll just ignore this test for now...